time/src/test/java/net/jqwik/api/time/MonthDayTests.java

```diff
@@ -0,0 +1,333 @@
+
+				Arbitrary<MonthDay> dates = Dates.monthDays().atTheLatest(monthDay);
+
+				assertAllGenerated(dates.generator(1000), random, md -> {
+					assertThat(md).isLessThanOrEqualTo(monthDay);
+					return true;
+				});
+
+			}
+
+			@Property
+			void between(@ForAll("monthDays") MonthDay startMonthDay, @ForAll("monthDays") MonthDay endMonthDay, @ForAll Random random) {
+
+				Assume.that(!startMonthDay.isAfter(endMonthDay));
+
+				Arbitrary<MonthDay> dates = Dates.monthDays().between(startMonthDay, endMonthDay);
+
+				assertAllGenerated(dates.generator(1000), random, md -> {
+					assertThat(md).isGreaterThanOrEqualTo(startMonthDay);
+					assertThat(md).isLessThanOrEqualTo(endMonthDay);
+					return true;
+				});
+			}
+
+			@Property
+			void betweenSame(@ForAll("monthDays") MonthDay monthDay, @ForAll Random random) {
+
+				Arbitrary<MonthDay> dates = Dates.monthDays().between(monthDay, monthDay);
+
+				assertAllGenerated(dates.generator(1000), random, md -> {
+					assertThat(md).isEqualTo(monthDay);
+					return true;
+				});
+
+			}
+
+		}
+
+		@Group
+		class MonthMethods {
+
+			@Property
+			void monthGreaterOrEqual(@ForAll("months") int month, @ForAll Random random) {
+
+				Arbitrary<MonthDay> dates = Dates.monthDays().monthGreaterOrEqual(month);
+
+				assertAllGenerated(dates.generator(1000), random, md -> {
+					assertThat(md.getMonth()).isGreaterThanOrEqualTo(Month.of(month));
+					return true;
+				});
+
+			}
+
+			@Property
+			void monthLessOrEqual(@ForAll("months") int month, @ForAll Random random) {
+
+				Arbitrary<MonthDay> dates = Dates.monthDays().monthLessOrEqual(month);
+
+				assertAllGenerated(dates.generator(1000), random, md -> {
+					assertThat(md.getMonth()).isLessThanOrEqualTo(Month.of(month));
+					return true;
+				});
+
+			}
+
+			@Property
+			void monthBetween(@ForAll("months") int startMonth, @ForAll("months") int endMonth, @ForAll Random random) {
+
+				Assume.that(startMonth <= endMonth);
+
+				Arbitrary<MonthDay> dates = Dates.monthDays().monthBetween(startMonth, endMonth);
+
+				assertAllGenerated(dates.generator(1000), random, md -> {
+					assertThat(md.getMonth()).isGreaterThanOrEqualTo(Month.of(startMonth));
+					assertThat(md.getMonth()).isLessThanOrEqualTo(Month.of(endMonth));
+					return true;
+				});
+
+			}
+
+			@Property
+			void monthBetweenSame(@ForAll("months") int month, @ForAll Random random) {
+
+				Arbitrary<MonthDay> dates = Dates.monthDays().monthBetween(month, month);
+
+				assertAllGenerated(dates.generator(1000), random, md -> {
+					assertThat(md.getMonth()).isEqualTo(Month.of(month));
+					return true;
+				});
+
+			}
+
+			@Property
+			void monthOnlyMonths(@ForAll("monthsOnlyMonths") Month[] months, @ForAll Random random) {
+
+				Arbitrary<MonthDay> dates = Dates.monthDays().onlyMonths(months);
+
+				assertAllGenerated(dates.generator(1000), random, md -> {
+					assertThat(md.getMonth()).isIn(months);
```

Comment:
This generates a vararg warning. Use List<Month> instead of Month[]. This happens in a few other places as well.

Reply:
I think the warning has now disappeared by using Set <month>.